staging/src/k8s.io/cli-runtime/pkg/printers/name.go

```diff
@@ -80,13 +83,21 @@ func (p *NamePrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 	}
 
 	name := "<unknown>"
+	namespace := ""
 	if acc, err := meta.Accessor(obj); err == nil {
 		if n := acc.GetName(); len(n) > 0 {
 			name = n
 		}
+		if p.WithNamespace {
+			if ns := acc.GetNamespace(); len(ns) > 0 {
+				namespace = ns
+			} else {
+				namespace = "<unknown>"
```

Comment:
There are cluster-scoped objects whose namespace is actually empty and I think it might be better to simply leave it empty.

Reply:
Thank you for your review. It makes sense so I've fixed that at [`45a69ab` (#94978)](https://github.com/kubernetes/kubernetes/pull/94978/commits/45a69ab7b93d60ec5fc98d2522f9801daaf9c263).